cylc/rose/__init__.py

```diff
@@ -164,4 +164,4 @@
 
 """
 
-__version__ = '0.2'
+__version__ = '0.3.0'
```

Comment:
Note: Not an accident, moving to a standard three part release number as per the milestone.